packages/grafana-data/src/types/datasource.ts

```diff
@@ -451,9 +451,15 @@ export interface DataQueryResponse {
 
   /**
    * Optionally include error info along with the response data
+   * @deprecated use errors instead
```

Comment:
```suggestion
   * @deprecated use errors instead -- will be removed in Grafana 10+
```
maybe

Reply:
Make sense if we're not seeing any usage of this moving forward. Are plugins checking/using this? How are we planning to display multiple errors for a panel title?